### ISSUE TITLE
Won't install

### ISSUE THREAD

**Issue by IceHacks** [meetalva/material-design#2]
The latest Alva Canary (v1.0.0-38) says that this is incompatible. Am I doing something wrong or is it just out of date?

This is what I did:

```shell
$ git clone https://github.com/meetalva/material-design.git

$ cd material-design

$ npm i

$ npm run build
```
That's all I did, nothing was changed except maybe when I built it.
There were no errors and the `./lib` folder looked fine.

and then I imported the `package.json` into Alva.

**Comment by marionebl** [comment]
Thanks for reporting, I can reproduce this. In the meantime you can use the npm installable version of `@metalva/material-design` to play with the componennts.

**Comment by IceHacks** [comment]
Ok, thank you

**Comment by IceHacks** [comment]
> In the meantime you can use the npm installable version of `@metalva/material-design` to play with the componennts.

This didn't work. Then I found out it needed to be `@meetalva` not `@metalva` lol. Thanks.

Edit:
Ok, finished installing. And said it was incompatible. I guess I'll make my own now.

**Comment by FunkMonkey** [comment]
I still have this problem with the newest Alva version, where I can install this designkit directly from inside Alva. After pressing "connect" it tries to install for ages and than complains that the library is incompatible...

**Comment by tilmx** [comment]
Yeah, we found out that there is some trouble with the Webpack build. We're investigating it at the moment...

**Comment by IceHacks** [comment]
Cool
